### DISCUSSION TITLE
Unable to create organization

### DISCUSSION THREAD

**Discussion by YuzukiTsuru** [community/community#21719] [New to GitHub]
<p><img src="https://user-images.githubusercontent.com/12003087/181066432-3d2a3724-c04a-4f18-b61a-80f9c6a1d0f9.png" alt="1.PNG" data-base62-sha1="pikLyUPsspFtpzb55PCiX3uWROU" width="222" height="237"></p>
<p>There is no problem with all the content, but there is no way to click the button to create the organization.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/f9914326752f375bb1b50d827eb8412ba3d0a3ac.png" data-download-href="/uploads/short-url/zBM717OljeBFOUaUuiFVBrU0pqs.png?dl=1" title="2.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12003087/181066433-b9c9bc7e-879d-4560-be51-c47c222828e9.png" alt="2.PNG" data-base62-sha1="zBM717OljeBFOUaUuiFVBrU0pqs" width="690" height="156" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/f9914326752f375bb1b50d827eb8412ba3d0a3ac_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2.PNG</span><span class="informations">695×158 4.54 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by seveas** [comment]
<p>I can only get that to happen if I tick the “this account is owned by a business” checkbox without filling in the name of the business. Can you check whether this is what happened?</p>

**Comment by YuzukiTsuru** [comment, marked as answer]
<p>I am not quite sure, I am completely normal now, it may be a network problem.</p>
